feat(vscode): add quick actions to status bar tooltip

Sysix opened the pull request:
> This PR enhances the VSCode extension's status bar functionality by adding quick action commands to the status bar tooltip. 

<img width="202" height="184" alt="grafik" src="https://github.com/user-attachments/assets/9816f28a-0a76-413f-9f43-4aca74182ac0" />
<img width="175" height="189" alt="grafik" src="https://github.com/user-attachments/assets/ffd76f3b-8b86-4508-a2f1-9a877c739d6f" />

Sysix commented:
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.com/github/pr/oxc-project/oxc/15962?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#15837** <a href="https://app.graphite.com/github/pr/oxc-project/oxc/15837?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#15962** <a href="https://app.graphite.com/github/pr/oxc-project/oxc/15962?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.com/github/pr/oxc-project/oxc/15962?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#15951** <a href="https://app.graphite.com/github/pr/oxc-project/oxc/15951?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#15950** <a href="https://app.graphite.com/github/pr/oxc-project/oxc/15950?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#15883** <a href="https://app.graphite.com/github/pr/oxc-project/oxc/15883?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* `main`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>0-merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.com/invite/github/oxc-project?ref=merge-queue-instructions-in-stack-comment&prId=15962).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

camc314 reviewed (approved):
nice! this is coolr!

Sysix committed:
feat(vscode): add quick actions to status bar tooltip (#15962)

> This PR enhances the VSCode extension's status bar functionality by adding quick action commands to the status bar tooltip.

<img width="202" height="184" alt="grafik" src="https://github.com/user-attachments/assets/9816f28a-0a76-413f-9f43-4aca74182ac0" />
<img width="175" height="189" alt="grafik" src="https://github.com/user-attachments/assets/ffd76f3b-8b86-4508-a2f1-9a877c739d6f" />
